.github/workflows/sbom.yml

```diff
@@ -144,21 +152,30 @@ jobs:
           echo '```' >> $GITHUB_STEP_SUMMARY
           echo "</details>" >> $GITHUB_STEP_SUMMARY
 
-      - name: Generate CycloneDX SBOM for Container Image
-        if: github.event_name != 'pull_request'
-        uses: anchore/sbom-action@v0
+      # Upload SBOMs as artifacts
+      - name: Upload SBOM Artifacts
+        uses: actions/upload-artifact@v4
         with:
-          image: ${{ steps.image_tag.outputs.IMAGE_TAG }}
-          artifact-name: opencost-container-sbom.cyclonedx.json
-          output-file: opencost-container-sbom.cyclonedx.json
-          format: cyclonedx-json
-
-      # Publish SBOMs to GitHub release (only for releases, not PRs)
+          name: sbom-files
+          path: |
+            opencost-source-sbom.spdx.json
+            opencost-source-sbom.cyclonedx.json
+            opencost-container-sbom.spdx.json
+            opencost-container-sbom.cyclonedx.json
+          if-no-files-found: ignore
+
+      # Attach SBOMs to GitHub release (only for releases, not PRs)
       - name: Attach SBOMs to GitHub Release
         if: github.event_name != 'pull_request'
-        uses: anchore/sbom-action/publish-sbom@v0
+        uses: softprops/action-gh-release@v1
         with:
-          sbom-artifact-match: ".*\\.spdx\\.json$|.*\\.cyclonedx\\.json$"
+          tag_name: v${{ steps.version_number.outputs.RELEASE_VERSION }}
```

Comment:
great, I like this a lot better, looks like less magic since we are telling it where exactly to upload. Reviewing the releases, it looks like our releases do in fact start with 'v' - I think RELEASE_VERSION logic strips that? might need to reconcile